.env-example

```diff
@@ -0,0 +1,2 @@
+DISCORD_WEBHOOK_URL=
```

Comment:
In an example you should write example variable in order for the others to understand what it should look like